native/src/utils/getNavigationDeepLinks.ts

```diff
@@ -1,13 +1,23 @@
-import type { Position } from 'geojson'
 import { Platform } from 'react-native'
 
+import { LocationModel } from 'api-client/src'
+
 /**
  * Depending on the platform this function returns the correct deeplink for navigation app
  * @param address{string}: location address
- * @param coordinates{number[]}: latitude,longitude
+ * @param location{number[]}: latitude,longitude
  */
-export const getNavigationDeepLinks = (address: string, coordinates: Position): string =>
-  Platform.select({
-    ios: `maps:${coordinates[1]},${coordinates[0]}?q=${address}`,
-    android: `geo:${coordinates[1]},${coordinates[0]}?q=${address}`
+export const getNavigationDeepLinks = (address: string, location: LocationModel): string | undefined => {
+  if (location.coordinates == null) {
+    return undefined
+  }
```

Comment:
```suggestion
export const getNavigationDeepLink = (location: LocationModel): string | null => {
  if (location.location && location.coordinates) {
    return null
  }
```

Reply:
```suggestion
export const getNavigationDeepLinks = (address: string, location: LocationModel): string | undefined => {
  if (!location.location && !location.coordinates) {
    return null
  }
```
Or am I missing something ^^

Reply:
Nearly, it should be an OR :grimacing: